yari-sys/src/yr_value.rs

```diff
@@ -23,6 +29,7 @@ pub enum YrValue {
     Dictionary(HashMap<String, YrValue>),
     Array(Vec<YrValue>),
     Structure(HashMap<String, YrValue>),
+    Reference(Option<Box<YrValue>>),
```

Comment:
I'm not sure if boxing YrValue is the best approach here, it doesn't seem right. I didn't come up with something better tho. If you've any tip, I'm happy to rework this part.

Reply:
fixed by getting rid of Reference type